client/src/views/Results.vue

```diff
@@ -312,6 +315,11 @@ export default defineComponent({
                     this.selectedPatentIndex--;
                     break;
             }
+
+            // turn highlight on node on. Timeout so to have the component react to state change
+            setTimeout(() => {
+                this.$store.commit('HIGHLIGHT_NODE_ON', this.selectedPatentIndex);
+            }, 100);
```

Comment:
Is the 100ms really necessary? I only ask because JavaScript is single threaded, and often just using set timeout will be enough to delay execution until after a whatever is executing is through. This might also be cleaner in an action, just in general.

Reply:
you're right, works also without the indicated ms! thanks for noting it!